microsoft-edge/devtools-guide-chromium/whats-new/2023/12/devtools-120.md

```diff
@@ -0,0 +1,88 @@
+---
+title: What's new in DevTools (Microsoft Edge 120)
+description: "Edge DevTools has a new user interface. Explain with Copilot in DevTools is generally available. New Recorder tool to record and replay common user flows and measure performance. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 12/07/2023
+---
+# What's New in DevTools (Microsoft Edge 120)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## DevTools has a new user interface
+
+DevTools has a new look, starting with Microsoft Edge 120!  The major design updates are the new **Activity Bar** with icons; **Quick View** in place of the Drawer at bottom of DevTools; and the ability to easily open, close, and move tools around:
+
+![Updated DevTools UI](./devtools-120-images/new-devtools-ui.png)
+
+See blog post: [Inside the all-new Edge DevTools user interface](https://blogs.windows.com/msedgedev/2023/12/20/inside-the-all-new-edge-devtools-ui/).
+
+If you prefer the legacy UI, you can switch back by going to DevTools > **Settings** > **Preferences** and selecting the **Switch back to the legacy DevTools UI** checkbox.  After reloading DevTools, the legacy UI appears.
+
+
+<!-- ====================================================================== -->
+## Explain with Copilot in DevTools is generally available
```

Comment:
confusing grammar in heading, b/c starts w/ name of feature as "Explain with Copilot in DevTools".
alts:
* The "Explain with Copilot in DevTools" feature is generally available
I might try that heading.

Reply:
done